VERSION

```diff
@@ -0,0 +1 @@
+v0.1.0
```

Comment:
Maybe add a `-dev` until we're ready to cut a release?

Reply:
Thanks, yep, a suffix with dev makes sense.

Reply:
Done